test/plugins/sync-intl.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const
+	expect = require('chai').expect,
+	sinon = require('sinon');
+
+require('chai')
+	.use(require('sinon-chai'))
+	.should();
+
+const
+	clientSyncIntl = require('../../src/plugins/sync-intl/client'),
+	hostSyncIntl = require('../../src/plugins/sync-intl/host');
+
+const MockClient = function() {};
+MockClient.prototype.request = function() {};
+
+const MockHost = function() {};
+MockHost.prototype.onRequest = function() {};
+
+describe('sync-intl', () => {
+
+	let setAttribute, hasAttribute, getAttribute;
+
+	beforeEach(() => {
+		setAttribute = sinon.stub();
+		hasAttribute = sinon.stub();
+		getAttribute = sinon.stub();
+		global.document = {
+			getElementsByTagName: function() {
+				return [{
+					getAttribute: getAttribute,
+					hasAttribute: hasAttribute,
+					setAttribute: setAttribute
+				}];
+			}
+		};
+	});
+
+	describe('client', () => {
+
+		let client, request;
+
+		beforeEach(() => {
+			client = new MockClient();
+			request = sinon.stub(client, 'request');
+		});
+
+		afterEach(() => {
+			request.restore();
```

Comment:
Doesn't really seem like a thing you need, given the client and the stub are also `*Each`